### ISSUE TITLE
brewfile gets rewritten, replacing ${HOSTNAME} with actual hostname

### ISSUE THREAD

**Issue by zcutlip** [rcmdnk/homebrew-file#93]
I have the following line at the end of my main Brewfile:

```
file ./${HOSTNAME}.Brewfile
```

When I run `brew file update`, my brewfile gets updated to say:

```
file ./osarian.Brewfile
```

Is this expected? I'd like it to keep the environment variable so I can use it generically on arbitrary hosts.

Homebrew & brew-file version:
```
Homebrew 2.5.1-6-g7589a35
Homebrew/homebrew-core (git revision 13c40; last commit 2020-09-14)
Homebrew/homebrew-cask (git revision dc938; last commit 2020-09-14)
brew-file v8.0.4 14/Sep/2020
```

**Comment by rcmdnk** [comment]
This is another bug introduced at v8.0.2, sorry.
It should be fixed at v8.0.5.

**Comment by zcutlip** [comment]
Awesome, it works now. Thank you!
